### DISCUSSION TITLE
Does the Nordstrom script work?

### DISCUSSION THREAD

**Discussion by CwnAnnwn** [alltheplaces/alltheplaces#3836] [General]
The site seems to have gone JS, which the script doesn't support.

**Comment by Cj-Malone** [comment]
You're right, it's broken and its been broken for a while.

They block non browser from accessing the website, it's a shame because they do put SEO stuff in there. They just don't let us read it.

Anyway, I've found an api so I'll fix it now.

**Reply by Cj-Malone** [reply]
#3837 opened.

**Reply by CwnAnnwn** [reply]
Thank you, that worked beautifully!
